pkg/report/writer.go

```diff
@@ -233,3 +235,34 @@ func (tw TemplateWriter) Write(results Results) error {
 	}
 	return nil
 }
+
+func toSarifRuleName(vulnerabilityType string) string {
+	var ruleName string
+	switch vulnerabilityType {
+	case vulnerability.Ubuntu, vulnerability.Alpine, vulnerability.RedHat, vulnerability.RedHatOVAL,
+		vulnerability.Debian, vulnerability.DebianOVAL, vulnerability.Fedora, vulnerability.Amazon,
+		vulnerability.OracleOVAL, vulnerability.SuseCVRF, vulnerability.OpenSuseCVRF, vulnerability.Photon,
+		vulnerability.CentOS:
+		ruleName = "Os Package Vulnerability "
+	case "npm", "yarn", "nuget", "pipenv", "poetry", "bundler", "cargo", "composer":
+		ruleName = "Programming Language Vulnerability "
+	default:
+		ruleName = "Other Vulnerability "
+	}
+	return ruleName + strings.Title(vulnerabilityType)
+}
+
+func toSarifErrorLevel(severity string) string {
+	var sarifErrorLevel string
```

Comment:
Instead of using a local variable, you just do the return within each of the case statements.

Reply:
fixed